starboard/build/config/base_configuration.gni

```diff
@@ -44,9 +44,6 @@ declare_args() {
   # Directory path to static contents' data.
   sb_static_contents_output_data_dir = "$root_out_dir/content/data"
 
-  # Whether this is a modular build.
```

Comment:
This is default, right? We should keep it.

Reply:
yes, readded